src/digitalIdentity-backend/src/test/java/didentity/amos/digitalIdentity/services/CredentialDefinitionServiceTest.java

```diff
@@ -0,0 +1,167 @@
+
+        String expected = "Succesfully created new credential definition.";
+        HttpStatus httpStatusExpected = HttpStatus.CREATED;
+
+        // Act
+        ResponseEntity<String> response = credentialDefinitionService.create(alias, comment, schemaId);
+        String responseAsString = response.getBody();
+        HttpStatus httpStatus = response.getStatusCode();
+
+        // Assert
+        assertEquals(expected, responseAsString);
+        assertEquals(httpStatusExpected, httpStatus);
+    }
+
+    @Test
+    void testCreate_withWrongParametersForCreateCredentialDefinition_shouldNotCreateCredentialAndReturn500() {
+        // Arrange
+        this.intallRessourceServiceMock_getDummyPng();
+
+        String alias = "test";
+        String comment = "test";
+        String schemaId = "test";
+
+        String expected = "Could not create a new credential.";
+        HttpStatus httpStatusExpected = HttpStatus.INTERNAL_SERVER_ERROR;
+
+        // Act
+        ResponseEntity<String> response = credentialDefinitionService.create(alias, comment, schemaId);
+        String responseAsString = response.getBody();
+        HttpStatus httpStatus = response.getStatusCode();
+
+        // Assert
+        assertEquals(expected, responseAsString);
+        assertEquals(httpStatusExpected, httpStatus);
+    }
+
+    @Test
+    void testCreate_WithMissingFile_ShouldNotFindFileAndReturn500() {
+        // Arrange
+        this.intallLissiApiServiceMock_createCredentialDefinition();
+
+        String alias = "test";
+        String comment = "test";
+        String schemaId = "test";
+
+        String expected = "Could not find file.";
+        HttpStatus httpStatusExpected = HttpStatus.INTERNAL_SERVER_ERROR;
+
+        // Act
+        ResponseEntity<String> response = credentialDefinitionService.create(alias, comment, schemaId);
+        String responseAsString = response.getBody();
+        HttpStatus httpStatus = response.getStatusCode();
+
+        // Assert
+        assertEquals(expected, responseAsString);
+        assertEquals(httpStatusExpected, httpStatus);
+    }
+
+    @Test
+    void testGetAllCredDefs_withValid_ShouldReturnAllCredentialDefinitionsAnd201() {
+        // Arrange
+        installLissiApiServiceMock_provideExistingCredDefs();
+
+        String activeState = "test";
+        String searchText = "test";
+
+        String expected = "All credential definitions should appear here.";
+        HttpStatus httpStatusExpected = HttpStatus.CREATED;
+
+        // Act
+        ResponseEntity<String> response = credentialDefinitionService.getAllCredDefs(activeState, searchText);
+        String responseAsString = response.getBody();
+        HttpStatus httpStatus = response.getStatusCode();
+
+        // Assert
+        assertEquals(expected, responseAsString);
+        assertEquals(httpStatusExpected, httpStatus);
+    }
+
+    @Test
+    void testGetAllCredDefs_withoutLissi_ShouldReturn500() {
+        // Arrange        
```

Comment:
See above. I think, you should be forcing a fail within a unit test with a mocked beahior